python/sdk_e2e/test_track_1/test_sas_rewewal.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Looks like these tests for reconnect enabled and reconnect disabled differ only in the `client_kwargs` fixture. Could you have a base class with the tests and then have two classes which inherit from it and override the `client_kwargs` fixture? It'd avoid repeating code.

Reply:
Good idea.  I think I'm going to refactor to a base class with common tests in a future change.